packages/wonder-blocks-data/util/request-handler.test.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+import RequestHandler from "./request-handler.js";
+
+describe("./request-handler.js", () => {
+    afterEach(() => {
+        /**
+         * This is needed or the JSON.stringify mocks need to be
+         * mockImplementationOnce. This is because if the snapshots need
+         * to update, they write the inline snapshot and that appears to invoke
+         * prettier which in turn, calls JSON.stringify. And if that mock
+         * throws, then boom. No snapshot update and a big old confusing test
+         * failure.
+         */
+        jest.restoreAllMocks();
```

Comment:
That's pretty scary.  Thanks for writing explanation of what goes on without this call to `restoreAllMocks()`